sdk/eventhub/Azure.Messaging.EventHubs/stress/stress-test-resources.bicep

```diff
@@ -0,0 +1,95 @@
+@description('The url suffix to use when creating storage connection strings.')
```

Comment:
`az bicep decompile -f <arm template>` is a great way to kickstart this.

Reply:
Huh, that's unfortunate, because all the normal deploy sub-commands in the CLI tools DO support native bicep templates (or at least do the conversion under the hood, though I doubt the direct API takes bicep).